source/includes/harvest/_approvals.md

```diff
@@ -190,3 +261,65 @@ type | 'job' or 'offer', where 'offer' returns only approvals which are for a sp
 | status | This is either "waiting" or "due" for pending approvals |
 | approver_group_id | This is Approver Group ID used in post requests to replace this user in approval steps. |
 
+## PUT: Replace an approver in an approver group
+
+```shell
+curl -X PUT 'https://harvest.greenhouse.io/v1/approver_groups/{id}/replace_approvers'
+  -H "On-Behalf-Of: {greenhouse user ID}"
+  -H "Authorization: Basic MGQwMzFkODIyN2VhZmE2MWRjMzc1YTZjMmUwNjdlMjQ6"
+```
+
+> The above command takes a JSON request, structured like this:
+
+```json
+{
+  "remove_user_id": 1443,
+  "add_user_id": 5432
+}
+```
+
+> The above returns a JSON success message with the following information.
+
+```
+{
+  "success": "true",
+  "message": "Approver group updated.",
+  "approval_flow_id": 454,
+  "approver_group_id": 343,
+  "removed_user_id": 1443,
+  "added_user_id": 5432,
+  "new_approver_id": 4332
+}
+```
+
+Removes the approver with user id given in remove_user_id and adds a new approver with the user id in add_user_id. Only approvers in unresolved groups may be replaced with this endpoint. If a group is currently "due", the new user will be notified that action is required.
+
+### HTTP Request
+
+`PUT https://harvest.greenhouse.io/v1/approver_groups/{approver_group_id}/replace_approvers`
+
+### Headers
+
+Header | Description
+--------- | -----------
+On-Behalf-Of | ID of the user issuing this request. Required for auditing purposes. Must have access to the approval flow.
+
+### URL Parameters
+
+Parameter | Description
+--------- | -----------
+id | The ID of the user whose pending approvals we want.
```

Comment:
How would this work? Don't you have to specify a specific approver group? So how could you further filter a single group down by a user?